examples/demo/src/visitors/SegmentsInput.tsx

```diff
@@ -1,18 +1,18 @@
 import * as React from 'react';
-import { useTranslate, SelectArrayInput, InputProps } from 'react-admin';
+import { useTranslate, SelectArrayInput, CommonInputProps } from 'react-admin';
```

Comment:
Why the rename of the type? It introduces a useless BC break

Reply:
It's not the same type. InputProps is from ra-core, this one contains UI and IMO ra-ui-material ui only props. It extends `InputProps`